src/lib/actions/profile/change-password.action.ts

```
@@ -0,0 +1,33 @@
+"use server";
+
+import { JSON_HEADER } from "@lib/constants/shared.constant";
+import { ChangePasswordRequest } from "@lib/types/profile/change-password";
+import { getDecodeToken } from "@lib/utils/get-decode-token";
+
+// Function to change the user's password
+export async function changePassword(data: ChangePasswordRequest) {
+  // Get token from decode token function
+  const token = await getDecodeToken() 
```

ESLint

[error] Missing semicolon.

(semi)